Add a mecanism to remove a temp file

Would it be possible to add something to delete the {package}-client.pot in the `local` of the package (if this file already exists), before creating it.

I quite often get the error here under and manually deleting the file resolves the problem, but I would prefer if the build system would take care of that....

```
WindowsError: [Error 183] Cannot create a file when that file already exists
CONST_Makefile:708: recipe for target 'sitn/locale/sitn-client.pot' failed
make: *** [sitn/locale/sitn-client.pot] Error 1
```

--- a/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl
+++ b/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl
@@ -696,6 +696,7 @@ $(PACKAGE)/locale/$(PACKAGE)-$(L10N_SERVER_POSTFIX).pot: \
 		$(VENV_BIN)/pot-create$(PYTHON_BIN_POSTFIX) \
 		$(SERVER_LOCALISATION_SOURCES_FILES)
 	$(PRERULE_CMD)
+	rm -f $@ # Because of WindowsError, thus only problematic on Windows
 	$(VENV_BIN)/pot-create --keyword _ --config $< --output $@ $(SERVER_LOCALISATION_SOURCES_FILES)
 	# removes the always changed date line
 	sed -i '/^"POT-Creation-Date: /d' $@
@@ -711,6 +712,7 @@ $(PACKAGE)/locale/$(PACKAGE)-$(L10N_CLIENT_POSTFIX).pot: \
 		.build/mapserver.timestamp \
 		.build/config.yaml
 	$(PRERULE_CMD)
+	rm -f $@ # Because of WindowsError, thus only problematic on Windows
 	$(VENV_BIN)/pot-create --config $< --output $@ $(I18N_SOURCE_FILES)
 	# removes the always changed date line
 	sed -i '/^"POT-Creation-Date: /d' $@

